content/blog/2020-01-ggplot2-3-3-0/index.Rmarkdown

```diff
@@ -0,0 +1,231 @@
+---
+title: "ggplot2 3.3.0"
+author: "Thomas Lin Pedersen"
+date: '2020-01-07'
+description: |
+  The next iteration of ggplot2 has just been released on CRAN, packed with new 
+  features and bug fixes. Read all about what's new here.
+photo:
+  author: Steve Harvey
+  url: https://unsplash.com/photos/xWiXi6wRLGo
+slug: ggplot2-3-3-0
+categories: ["package"]
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>"
+)
+```
+
+We're so happy to announce the release of ggplot2 3.3.0 on CRAN. ggplot2 is a system for declaratively creating graphics, based on *The Grammar of Graphics*. You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.
+
+This release is a substantial release with both big new features and internal rewrites. It contains most of the work Dewey Dunnington did as part of his internship at RStudio, along with contributions from many others. Dewey will continue working on ggplot2 as part of the core team, and we are very lucky to have him.
+
+## New features
+The last ggplot2 release was mainly a performance-oriented release. This one, on the other hand is packed with features, big and small. There is also a slew of bug-fixes, but as that make for less interesting reading, please consult the NEWS.md file for those. Further, we have now removed reshape2 from the dependencies, which further removes the indirect dependencies on plyr, stringi and stringr. We have been shaving off dependencies in ggplot2 over a range of releases, and with the removal of reshape2 we are now close to being as lean as we believe is possible.
+
+### Rewrite of axis code
+While Dewey has worked on a lot of different parts of the ggplot2 code base, the lion share has been concerned with a rewrite of the positional guide (axis) internals. While, at a high level, axes and legends are equivalent (they are both guides used for scales), this has not been true for the underlying code. With this release we have streamlined the implementation considerably, and paved the way for a full guide rewrite in a future release (guides are one of the last part waiting to be updated to ggproto). Apart from making our life as ggplot2 developers easier, the rewrite also comes with a slew of user facing improvements and features. All of this is contained in the new `guide_axis()` function that works equivalently to e.g. `guide_legend()`.
```

Comment:
```suggestion
While Dewey has worked on a lot of different parts of the ggplot2 code base, the lion's share has been concerned with a rewrite of the positional-guide (axis) internals. While, at a high level, axes and legends are equivalent (they are both guides used for scales), this has not been true for the underlying code. With this release, we have streamlined the implementation considerably, and paved the way for a full guide rewrite in a future release (guides are one of the last part waiting to be updated to ggproto). Apart from making our life as ggplot2 developers easier, the rewrite also comes with a slew of user-facing improvements and features. All of this is contained in the new `guide_axis()` function that works equivalently to e.g. `guide_legend()`.
```

Reply:
^^ changed lion share to lion's share; added hyphens to compound modifiers (e.g. user facing improvements → user-facing improvements)

Also, might link to the `guide_axis()` function reference, once the main pkgdown site is a go.